AdvLoggerPkg/Library/BasePanicLibAdvancedLogger/BasePanicLibAdvancedLogger.c

```diff
@@ -0,0 +1,61 @@
+/** @file
+  An instance of the Panic Library that outputs to advanced logger.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Base.h>
+#include <Library/BaseLib.h>
+#include <Library/PrintLib.h>
+#include <Library/AdvancedLoggerLib.h>
+#include <Library/PanicLib.h>
+
+//
+// Define the maximum panic message length that this library supports
+//
+#define MAX_PANIC_MESSAGE_LENGTH  0x100

```

Comment:
nit:

AdvLoggerPkg\Library\BaseDebugLibAdvancedLogger\DebugLib.c defines MAX_DEBUG_MESSAGE_LENGTH  (as 0x100 also)

Do you think it would be worth unifying this define somewhere more public?

Reply:
I noticed that too. Here's the places in `AdvLoggerPkg` that set the `MAX_..._MESSAGE_LENGTH` to `0x100`:

- AdvLoggerPkg/Library/AssertLib/AssertLib.c
- AdvLoggerPkg/Library/AssertTelemetryLib/AssertLib.c
- AdvLoggerPkg/Library/BaseDebugLibAdvancedLogger/DebugLib.c
- AdvLoggerPkg/Library/BasePanicLibAdvancedLogger/BasePanicLibAdvancedLogger.c

I think the `AssertLib` instances could be consolidated. For the others, I think this is an arbitrary but commonly chosen value. If we conclude the tradeoff of making this the same across all instances with a common definition versus the flexibility to easily change an individual instance value is worth it, I'll update it.

I'm fairly neutral on the change itself, but I'm leaning toward doing it in another PR (if we choose to) since it will affect more code than just the `PanicLib` instance in the scope of this PR.

Reply:
I'm going to resolve this for this PR and we can discuss further if consolidation makes sense in a future change.